Common/src/Pollster/AgentIdentifier.cs

```diff
@@ -0,0 +1,28 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2024. All rights reserved.
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY, without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Net;
+
+using ArmoniK.Core.Common.Utils;
+
+namespace ArmoniK.Core.Common.Pollster;
+
+public static class AgentIdentifier
+{
+  public static readonly string OwnerPodId   = LocalIpFinder.LocalIpv4Address();
+  public static readonly string OwnerPodName = Dns.GetHostName();
```

Comment:
What if those throw on the control plane?

Reply:
Why should they ?

Reply:
I created a non-static class